index.js

```diff
@@ -66,6 +66,11 @@ const recommended = {
   },
 };
 
-Object.assign(plugin.configs, { recommended });
+const recommendedLegacy = {
+  plugins: ['security'],
+  rules: recommended.rules,
+};
+
+Object.assign(plugin.configs, { recommended, 'recommended-legacy': recommendedLegacy });
```

Comment:
To improve readability
```suggestion
Object.assign(plugin.configs, {
  recommended,
  'recommended-legacy': recommendedLegacy
});
```